cmd/influx/cli/cli.go

```diff
@@ -211,12 +211,8 @@ func (c *CommandLine) Run() error {
 	c.Version()
 
 	if c.Type == QueryLanguageFlux {
-		repl, err := getFluxREPL(c.URL, c.ClientConfig.Username, c.ClientConfig.Password)
-		if err != nil {
-			return err
-		}
-		repl.Run()
-		os.Exit(0)
+		// TODO(lesam): steal 2.x flux client
+		return fmt.Errorf("ERROR: flux repl missing due to flux upgrade")
```

Comment:
Note these errors - this will be a pretty major change to the `influx` client for 1.x flux queries, as we will be removing the REPL. We should obviously fix this TODO in the next week or two, before 1.9 releases.

Reply:
```

Dockerfile_build_ubuntu32

```

Reply:
@lesam 

Dockerfile_build_ubuntu32